packages/mdc-textfield/README.md

```diff
@@ -149,210 +135,100 @@ See [here](icon/) for more information on using icons.
-| deregisterInputInteractionHandler(evtType: string, handler: EventListener) => void | Deregisters an event listener on the native input element for a given event |
-| registerBottomLineEventHandler(evtType: string, handler: EventListener) => void | Registers an event listener on the bottom line element for a given event |
-| deregisterBottomLineEventHandler(evtType: string, handler: EventListener) => void | Deregisters an event listener on the bottom line element for a given event |
-| getNativeInput() => {value: string, disabled: boolean, badInput: boolean, checkValidity: () => boolean}? | Returns an object representing the native text input element, with a similar API shape. The object returned should include the `value`, `disabled` and `badInput` properties, as well as the `checkValidity()` function. We _never_ alter the value within our code, however we _do_ update the disabled property, so if you choose to duck-type the return value for this method in your implementation it's important to keep this in mind. Also note that this method can return null, which the foundation will handle gracefully. |
-
-MDC Text Field has multiple optional sub-elements: bottom line and helper text. The foundations of these sub-elements must be passed in as constructor arguments for the `MDCTextField` foundation. Since the `MDCTextField` component takes care of creating its foundation, we need to pass sub-element foundations through the `MDCTextField` component. This is typically done in the component's implementation of `getDefaultFoundation()`.
-
-#### The full foundation API
-
-##### MDCTextFieldFoundation.isDisabled() => boolean
-
-Returns a boolean specifying whether or not the input is disabled.
-
-##### MDCTextFieldFoundation.setDisabled(disabled: boolean)
-
-Updates the input's disabled state.
-
-##### MDCTextFieldFoundation.setValid(isValid: boolean)
-
-Sets the validity state of the Text Field. Triggers custom validity checking.
-
-##### MDCTextFieldFoundation.handleTextFieldInteraction(evt: Event)
-
-Handles click and keydown events originating from inside the Text Field component.
-
-##### MDCTextFieldFoundation.activateFocus()
-
-Activates the focus state of the Text Field. Normally called in response to the input focus event.
-
-##### MDCTextFieldFoundation.deactivateFocus()
-
-Deactivates the focus state of the Text Field. Normally called in response to the input blur event.
-
-##### MDCTextFieldFoundation.handleBottomLineAnimationEnd(evt: Event)
-
-Handles the end of the bottom line animation, performing actions that must wait for animations to
-finish. Expects a transition-end event.
-
-##### MDCTextFieldFoundation.setHelperTextContent(content)
-
-Sets the content of the helper text, if it exists.
-
-### Theming
-
-MDC Text Field components use the configured theme's primary color for its underline and label text
-when the input is focused.
-
-MDC Text Field components support dark themes.
+### `MDCTextFieldAdapter`
```

Comment:
Not sure if we want to add it here or in the future, but it'd be nice to separate what's useful for someone who wants to use MDC (i.e. all the stuff above this), or someone who wants to build a wrapper around MDC (i.e foundation/adapter implementation).

Reply:
I think we should think about this for the future. Right now we only have one README per component, so we put this all into one README.